tools/doc/README.md

```diff
@@ -106,3 +106,5 @@ This event is emitted on instances of `SomeClass`, not on the module itself.
 * Methods have (list of arguments, returned value if defined, description).
 * Modules have (description, Properties, Functions, Classes, Examples).
 * Properties have (type, description).
+
+Every Function and Method should have an in-line example showing their use.
```

Comment:
I am not sure if this is the best location to document goals. Also, I think this might be too broad.

Reply:
@tniessen I looked at either this doc or https://github.com/nodejs/node/tree/master/doc and this felt was the better of the 2. I agree neither is a great fit.  Another option might be to add a `maintaining-api-docs.md` guide in https://github.com/nodejs/node/tree/master/doc/contributing. Do you have any other suggestions?

When you say too broad, do you mean that having an in-line example for every function and method does not make sense or that the description needs to be more specific in some way?

Reply:
```suggestion
Every function should have an inline usage example.
```